aten/src/ATen/native/native_functions.yaml

```diff
@@ -1600,6 +1600,11 @@
 
 - func: celu_(Tensor(a!) self, Scalar alpha=1.0) -> Tensor(a!)
 
+- func: mkldnn_sigmoid(Tensor self) -> Tensor
+  dispatch:
+    MkldnnCPU: mkldnn_sigmoid
```

Comment:
can't you put this dispatch directly on `sigmoid`? It does have an entry in Derivatives.yaml so it should work as a direct dispatch already happens there

Reply:
Sure will try